test/integration/opaqueports/testdata/opaque_ports_client.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: slow-cooker-opaque-pod
+spec:
+  template:
+    metadata:
+      annotations:
+        linkerd.io/inject: "enabled"
+      labels:
+        app: slow-cooker-opaque-pod
+    spec:
+      containers:
+      - name: slow-cooker-opaque-pod
+        image: buoyantio/slow_cooker:1.3.0
+        command:
+        - "/bin/sh"
```

Comment:
any reason to continue invoking this through `sh -c` instead of just invoking slow_cooker directly? (and similarly throughout?)

Reply:
Good spot! No reason at all, added container args instead.